### DISCUSSION TITLE
Add support for partitioned cookies (CHIPS)

### DISCUSSION THREAD

**Discussion by Muosvr** [vercel/next.js#63932] [Ideas]
### Goals

1. Add support for the `partitioned` attribute in cookies
2.
3.

### Non-Goals

1.
2.
3.

### Background

Chrome has started to [deprecate third party cookies](https://developers.google.com/privacy-sandbox/3pcd), this leaves iframe application unable to persist state for purposes such as authentication. In order for iframe applications to continue to use cookies without requiring [additional user interaction](https://developers.google.com/privacy-sandbox/3pcd/storage-access-api), they need to set the `partitioned` attribute on the cookie.

### Proposal

Add `partitioned` attribute to ResponseCookie and the ability to set `partitioned` in Set-Cookie header

**Comment by xarielah** [comment]
Seems like it exists if you set the cookie using the Next.js built in cookie function.
I seem to experience issues with my presumed third-party API which supplies me with token cookies though and I cannot seem to access the token cookies using the normal cookie function.
